UPDATE DOI

silkroadnomad opened the pull request:
added derivation_path to DOI (Doichain)

cipig commented:
https://github.com/satoshilabs/slips/pull/1819

cipig commented:
@silkroadnomad if you want to replace icon, you need to replace the one in icons_original and delete the one in icons, so it is regenerated by the GH action script... but i saw that the icon was anyway the same, so no need to replace... and it also needs a transparent background

silkroadnomad commented:
I made a new PR since there was a bot PR which accidentally confused the old PR.
https://github.com/KomodoPlatform/coins/compare/master...silkroadnomad:coins:master
I added doi.png to icons_original and deleted doi.png in icons. Sorry for the inconvinience. 
https://github.com/KomodoPlatform/coins/pull/1167
